sentry/src/main/java/io/sentry/SentryEvent.java

```diff
@@ -11,8 +11,7 @@
 import org.jetbrains.annotations.Nullable;
 import org.jetbrains.annotations.TestOnly;
 
-public final class SentryEvent implements IUnknownPropertiesConsumer {
-  private SentryId eventId;
+public final class SentryEvent extends SentryItem implements IUnknownPropertiesConsumer {
```

Comment:
I'm not against inheritance but some people have strong feelings about it (inheritance/composition tabu).
if the idea is to move all common fields to `SentryItem` so we could have a `Transaction` class with specific items that don't fit into SentryEvent, I'm all for it.

Reply:
Yes that's exactly the goal

Reply:
Can we call it `SentryEventBase` or something else? I think we can do better than `SentryItem`.

Today this thing is a _error event_, a _transaction event_.. I got nothing.

Reply:
Also note it's a bit more complicated because Transaction has `contexts.trace` but possibly (not sure now) all the other `contexts` known types.
While error event has what it does, and it won't have that `contexts.trace`. Which means we might need some common `Contexts`, possibly with a generic SentryBase<TContexts> and a generic constraint on the Contexts. Sounds hacky as I write it.

Reply:
I like the `SentryEventBase`. Will see about the contexts once I get there.